xml/OptionPanel.xml

```diff
@@ -101,6 +110,30 @@
             </Cell>
           </Row>
 
+          <!-- Option: card language -->
+          <Row class="option-text">
+            <Cell class="option-dropdowntext">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Card language</Text>
+                <Text class="description">Downloading a supported campaign or importing a deck will use this language for the cards.</Text>
```

Comment:
Might be worth pointing out that cards already loaded won't be affected, but not sure how much room there is to work with

Reply:
I can't think of a concise way to convey this. We only have two lines 😓